package.json

```diff
@@ -21,11 +21,12 @@
     "@emotion/react": "^11.9.0",
     "@emotion/styled": "^11.9.0",
     "@faker-js/faker": "^8.4.1",
+    "@next-auth/prisma-adapter": "^1.0.7",
     "@prisma/client": "^5.17.0",
     "dotenv": "^16.4.5",
     "framer-motion": "^6.3.0",
     "next": "^14.2.13",
-    "next-auth": "^5.0.0-beta.19",
+    "next-auth": "^4.24.8",
```

Comment:
Does the next-auth version need to change?

Reply:
Yes